hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/wal/FSHLog.java

```diff
@@ -393,6 +393,8 @@ protected void doReplaceWriter(Path oldPath, Path newPath, Writer nextWriter) th
             inflightWALClosures.remove(oldPath.getName());
             if (!isUnflushedEntries()) {
               markClosedAndClean(oldPath);
+            } else {
+              LOG.debug("WAL has unflushed entries path: " + oldPath);
```

Comment:
Should you add some metrics to track this event? Is this PR about adding logs to help debugging? is there more to the fix?